iis-core/src/test/java/eu/dnetlib/iis/core/AbstractOozieWorkflowTestCase.java

```diff
@@ -233,4 +317,35 @@ private void printOozieJobLog(String jobId) {
 			log.warn("Unable to check oozie job log");
 		}
 	}
+	
+	private void importFilesFromHdfs(WorkflowTestResult result, List<String> filesPaths, String workingDir) {
```

Comment:
In my opinion it would be better to have the import methods in external service class.
In addition I would change the methods... we'll talk about it.